sdcm/cluster_aws.py

```diff
@@ -648,9 +649,8 @@ def destroy(self):
         self.stop_task_threads()
         self._instance.terminate()
         if self.eip_allocation_id:
-            client = boto3.client('ec2', region_name=self.parent_cluster.region_names[self.dc_idx])
-            response = client.release_address(AllocationId=self.eip_allocation_id)
-            self.log.debug("release elastic ip . Result: %s\n", response)
```

Comment:
Why the log was removed?

Reply:
since this function doesn't return value...